orchestrate.py

```diff
@@ -69,13 +69,16 @@ def get(self, path=None):
         self.render("loading.html", path=path)
 
 
-class StatsHandler(BaseHandler):
+class APIStatsHandler(BaseHandler):
     def get(self):
         '''Returns some statistics/metadata about the tmpnb server'''
         response = {
                 'available': len(self.pool.available),
                 'capacity': self.pool.capacity,
-                'version': '0.0.1-dev',
+                'version': '0.1.0',
+                'container_image': self.pool.container_config.image,
+                'docker_version': self.spawner.docker_client._docker_client.version(),
+                'docker_containers': self.spawner.docker_client._docker_client.containers()
```

Comment:
Yeah that's ok. It's only this call to the number of containers and likely only affects the larger deployments (>500 users/containers in the pool). This number is also likely to be redundant as the number of containers equals the capacity + is tmpnb containerized + is the proxy containerized + any another containers.

I'd keep the capacity and available metrics instead.